Dockerfile

```diff
@@ -1,56 +1,129 @@
-            ninja-build                                     \
-            python3-pip                                     \
-            python3.8-dev                                   \
-            libgraphviz-dev                                 \
-            libjpeg-dev                                     \
-            graphviz                                        \
-            vim                                             \
-            file                                            \
-            gdb                                             \
-            libncurses5-dev                                 \
-            apt-transport-https                             \
-            ca-certificates                                 \
-            libstdc++-10-dev
+
+ARG BUILD_TYPE="Release"
+ARG PARALLEL_LINK_JOBS=1
+ARG LLVM_TARGET_NAME=X86
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
+  ninja-build                                         \
+  python3-pip                                         \
+  python3.8-dev                                       \
+  python-is-python3                                   \
+  ca-certificates                                     \
+  libstdc++-10-dev                                    \
+  golang                                              \
+  clang-12                                            \
+  cmake                                               \
+  git                                                 \
+  file
+
+RUN update-alternatives --install /usr/bin/opt opt /usr/bin/opt-12 10
+RUN update-alternatives --install /usr/bin/llvm-link llvm-link /usr/bin/llvm-link-12 10
+RUN update-alternatives --install /usr/bin/llvm-ar llvm-ar /usr/bin/llvm-ar-12 10
+RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-12 10
+RUN update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-12 10
+
+WORKDIR /
+RUN git clone --depth 1 --branch llvmorg-13.0.0 https://github.com/llvm/llvm-project.git
+RUN git clone --depth 1 --branch master https://github.com/trailofbits/blight.git
+
+RUN pip3 install pytest /blight
 
 RUN update-ca-certificates
 
-RUN apt-get install -y software-properties-common && \
-      # deadsnakes PPA allows us to install python3.10 on focal
-      add-apt-repository ppa:deadsnakes/ppa && \
-      apt-get -y update && \
-      apt-get install -y \
-            python3.10-dev \
-            # make update-alternatives work
-            python3.10-distutils \
-            curl && \
-      # need a pip compatible with 3.10
-      curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10 && \
-      update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.10 10 && \
-      python3 -m pip install pytest
-
-WORKDIR /blight
-RUN git clone https://github.com/trailofbits/blight.git .
-RUN pip3 install .
+RUN GO111MODULE=off go get github.com/SRI-CSL/gllvm/cmd/...
```

Comment:
Thanks for updating the gllvm! I may have missed it, but what golang version is supported in this new Dockerfile version? I think the current `master` Polytracker base Dockerfile is pinned to an old Golang as well.